### DISCUSSION TITLE
在taro中使用选项卡segment的时候segment item的key没有用

### DISCUSSION THREAD

**Discussion by ckdroid** [TaleLin/lin-ui#1265] [求助] [answered]
```
  render() {
    return (
      <l-segment activeKey="one" onLinchange={this.changeTabs}>
        <l-segment-item tab="客厅" key="one" />
        <l-segment-item tab="卧室" key="two" />
        <l-segment-item tab="厨房" key="three" />
    </l-segment>   
    )
  }
```

![122](https://user-images.githubusercontent.com/8552889/104729682-c1d8d280-5773-11eb-9e5b-f127fe88edd7.PNG)
![123](https://user-images.githubusercontent.com/8552889/104729707-cb623a80-5773-11eb-9b91-867b3504b4e9.PNG)

**Comment by juzi214032** [comment, marked as answer]
抱歉 lin-ui 不对第三方框架做适配，这应该是 taro 的问题，可以给他们提 issue 解决
